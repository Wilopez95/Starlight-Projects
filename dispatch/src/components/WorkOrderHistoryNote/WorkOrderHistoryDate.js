import moment from 'moment';
import PropTypes from 'prop-types';

const WorkOrderHistoryDate = (props) => {
  const { createdDate } = props;
  return <p>{moment(createdDate).format('MM/DD/YYYY')}</p>;
};

WorkOrderHistoryDate.propTypes = {
  createdDate: PropTypes.string.isRequired,
};
export default WorkOrderHistoryDate;
