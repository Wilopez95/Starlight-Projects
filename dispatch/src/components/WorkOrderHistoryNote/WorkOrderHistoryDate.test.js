import { shallow, mount } from 'enzyme';
import WorkOrderHistoryDate from './WorkOrderHistoryDate';

const testDate = '2019-08-25T00:53:45.680Z';

describe('<WorkOrderHistoryDate />', () => {
  it('should render correctly with props', () => {
    const component = shallow(<WorkOrderHistoryDate createdDate={testDate} />);

    expect(component).toMatchSnapshot();
  });
  it('should render and display the date', () => {
    const wrapper = mount(<WorkOrderHistoryDate createdDate={testDate} />);
    expect(wrapper.props().createdDate).toEqual(testDate);

    expect(wrapper.text()).toContain('08');
  });
});
