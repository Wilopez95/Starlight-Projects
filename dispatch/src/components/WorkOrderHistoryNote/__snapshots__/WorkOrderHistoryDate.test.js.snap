// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<WorkOrderHistoryDate /> should render correctly with props 1`] = `ShallowWrapper {}`;
